code/game/machinery/computer/HolodeckControl.dm

```diff
@@ -383,9 +383,27 @@
 /obj/structure/rack/holorack
 	can_deconstruct = FALSE
 
+
+/obj/structure/stool/holostool
+	can_deconstruct = FALSE
+
+
 /obj/item/holo
 	damtype = STAMINA
 
+/obj/item/stool/holostool
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
Item/Holostool is a subtype of item/stool,  there's no need to specify its w_class which it should inherits from its parent (WEIGHT_CLASS_HUGE), why would a holo one becomes less bulky anyway?

Reply:
That's how the holostool was defined originally. I didn't check that beforehand. I'll look it over